src/main/java/teammates/ui/controller/Action.java

```diff
@@ -452,12 +453,39 @@ public String getRequestParamValue(String paramName) {
     }
     
     /**
+     * Retrieves the value for the specified parameter expected to be present in the http request.
+     *
+     * @param paramName  a constant from the {@link Const.ParamsNames} class.
+     * @return the value of the specified parameter if it exists, asserts otherwise.
+     */
+    public String getExpectedRequestParamValue(String paramName) {
+        String value = HttpRequestHelper.getValueFromParamMap(requestParameters, paramName);
+        Assumption.assertPostParamNotNull(paramName, value);
+        return value;
+    }
+    
+    /**
      * @return null if the specified parameter was not found in the request.
      */
     public String[] getRequestParamValues(String paramName) {
         return HttpRequestHelper.getValuesFromParamMap(requestParameters, paramName);
     }
     
+    /**
+     * Retrieves the values for the specified parameter expected to be present in the http request.
+     *
+     * @param paramName  a constant from the {@link Const.ParamsNames} class.
+     * @return the values of the specified parameter if it exists, asserts otherwise.
+     */
+    public String[] getExpectedRequestParamValues(String paramName) {
+        String[] values = HttpRequestHelper.getValuesFromParamMap(requestParameters, paramName);
+        if (values == null) {
+            throw new NullPostParameterException(String.format(Const.StatusCodes.NULL_POST_PARAMETER,
+                    paramName));
+        }
+        return values;
+    }
+    
```

Comment:
I like making the method generic as well, would also like input from more senior developers in case there's a reason not to do so.